Bottom: 238233c591013631921cdc4eb83b1f7ee6ffe8b8
Top:    ac7e6d7819eb42a8951d5e17004929114ce7d8c7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 19:23:27 +0100

Fix outline drawing to only apply on copper layers

This matches comments in the code, and fixes (removes) outline drawing
on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f

"Fix PS rect/outline logic"


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 7f2ae8b..1f25448 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1001,10 +1001,11 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") != 0 &&
-      strcmp (name, "route") != 0
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
